solidity/contracts/Staking.sol

```diff
@@ -41,19 +41,26 @@ contract Staking {
         uint256 stakeBuyinRate;  
     }
 
+    /**
+     * @dev Structure to store Rewards details.
+     * Contains the address of the rewards tokens and amount of rewards distribution
+     */
+    struct RewardsTokens {
+        address token;
+        uint256 totalReward; 
+    }
+
+    // A dynamically-sized array of `RewardsTokens` structs.
+    RewardsTokens[] public RewardsTokens;
```

Comment:
The `RewardsTokens` struct and variable got the same name. by convention var names should not be in capital

Reply:
@leonprou, I will update the tests and correct the name of the structs and modify the amount of interest according to the token.